.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,47 @@
+name: enclave-cc e2e tests
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator enclave-cc install e2e tests
+    runs-on: ubuntu-latest
```

Comment:
Could you set a specific version of Ubuntu here?

We would like to avoid hitting issues on the tests due to the "host OS" changing something not expected.

Reply:
thanks, fixed.